src/Google/Ads/GoogleAds/Lib/V1/GoogleAdsClientBuilder.php

```diff
@@ -112,10 +112,10 @@ public function withDeveloperToken($developerToken)
      * create a separate GoogleAdsClient instance for each manager account. Use this method to
      * set each login customer ID and call build() to create a separate instance.
      *
-     * @param int $loginCustomerId the login customer ID
+     * @param string|null $loginCustomerId the login customer ID
      * @return self this builder
      */
-    public function withLoginCustomerId($loginCustomerId)
+    public function withLoginCustomerId(?string $loginCustomerId)
```

Comment:
Let be consistent with the above too.
If you want to use ?string, please modify the above V0 to be the same.

Reply:
What do you mean? This change has been made on both the V0 and V1 classes.

Reply:
Sorry, I meant "fromFile()" method.

Reply:
I think they are two different cases because fromFile can be called with no arguments (as it is called in the examples) and in that case it sets a default value. 

In my opinion, withLoginCustomerId shouldn't be called with no arguments because we don't provide a default value, so it's correct to have users call it explicitly passing null if they want to set it to null.

To clarify, calling `fromFile()` is acceptable in my opinion, while calling `withLoginCustomerId()` isn't, because we don't set a default value and if users want to unset it they should be explicitly be calling `withLoginCustomerId(null)`.

WDYT?

Reply:
Got it. This makes sense to me, as we don't expect users to call fromFile(null), so having the signature as fromFile(?string $fileName = null) wouldn't make sense too.